pytential/symbolic/execution.py

```diff
@@ -229,17 +231,37 @@ def map_inverse(self, expr):
     def map_interpolation(self, expr):
         operand = self.rec(expr.operand)
 
-        if isinstance(operand, cl.array.Array):
-            from pytential.symbolic.dof_connection import connection_from_dds
+        if isinstance(operand, (cl.array.Array, list)):
+            conn = self.places.get_connection(expr.from_dd, expr.to_dd)
 
-            conn = connection_from_dds(self.places,
-                    expr.from_dd, expr.to_dd)
-            return conn(self.queue, operand).with_queue(self.queue)
+            if isinstance(operand, list):
+                return conn(self.queue, operand)
+            else:
+                return conn(self.queue, operand).with_queue(self.queue)
```

Comment:
Mostly because of [this bug](https://gitlab.tiker.net/inducer/meshmode/merge_requests/60), which should be fixed now. I'm going to remove the second `with_queue`, since no one else is doing that.